unstructured/partition/md.py

```diff
@@ -23,20 +24,17 @@ def partition_md(
     include_metadata: bool = True,
     parser: VALID_PARSERS = None,
 ) -> List[Element]:
-    if not any([filename, file, text, url]):
-        raise ValueError("One of filename, file, or text must be specified.")
+    # Verify that only one of the arguments was provided
+    exactly_one([filename, file, text, url], ["filename", "file", "text", "url"])
 
-    if filename is not None and not file and not text and not url:
+    if filename:
```

Comment:
```suggestion
    if filename is not None:
```
Similar to my other comments, if `filename == ""` and `file`, `text`, and `url` are `None`, it will bypass all the clauses in the `if` statement and `text` will be undefined in line 48.